src/CMake/PluginVsInstall.cmake.in

```diff
@@ -201,6 +204,8 @@ set(VISIT_JAVA                   @VISIT_JAVA@)
 set(VISIT_SLIVR                  @VISIT_SLIVR@)
 set(HAVE_LIBOSPRAY               @HAVE_LIBOSPRAY@)
 set(HAVE_OSPRAY                  @HAVE_OSPRAY@)
+set(HAVE_LIBANARI                @HAVE_LIBANARI@)
```

Comment:
AFAICT, I see `#ifdef HAVE_ANARI` used every where in C source code. So, I don't know what `HAVE_LIBANARI` is used for. Can you maybe clarify a bit.

Reply:
Just copying OSPRay here. My assumption is OSPRay was doing it for a reason and was vetted before the PR was merged so I also did it for ANARI.